script/glam/backfill_glean_all_fenix

```diff
@@ -22,6 +22,7 @@ EOD
 export DATASET=${DATASET:-glam_etl_dev}
 START_DATE=${START_DATE?}
 END_DATE=${END_DATE?}
+RUN_EXPORT=${RUN_EXPORT:-true}
 
 dir="$(dirname "$0")"
 
```

Comment:
Can this script run `backfill_glean` instead of `run_glam_sql`?  Then it wouldn't need the `ds_range` definition.  Is there a need to run the daily then incremental for one day at a time that I'm forgetting?